src/components/TopBar.tsx

```diff
@@ -1,10 +1,14 @@
-import { AppBar } from "@mui/material"
+import { AppBar, Box, Toolbar, Typography } from "@mui/material"
+import { PropsWithChildren } from "react"
 import LogoutButton from "./LogoutButton"
 
-export const TopBar = () => {
+export const TopBar = ({ show_logout }: PropsWithChildren<{ show_logout: boolean }>) => {
   return (
-    <AppBar sx={{ width: 1, display: "flex", position: "sticky", bgcolor: "white" }}>
-      <LogoutButton />
+    <AppBar color="secondary" sx={{ width: 1, display: "flex", position: "sticky" }}>
+      <Toolbar>
+        <Typography sx={{ flexGrow: 1 }} variant="h6">Smart Events</Typography>
+        {show_logout && <LogoutButton />}
```

Comment:
Why put the show_logout boolean in an && statement with the LogoutButton rather than if-statement (as done in many of the previous functions) to determine whether it should be shown or not?